pkgs/development/python-modules/lpc-checksum/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, poetry-core
+, intelhex
+}:
+
+buildPythonPackage rec {
+  pname = "lpc-checksum";
+  version = "3.0.0";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
Is it possible to use the [source repo](https://github.com/basilfx/lpc_checksum)? There are tests there and they should be working normally by adding `pytestCheckHook`.

Reply:
Fixed.